contrib/gitian-build.py

```diff
@@ -161,6 +167,11 @@ def main():
         os.environ['USE_DOCKER'] = '1'
     elif not args.kvm:
         os.environ['USE_LXC'] = '1'
+        if args.is_bionic:
+            if not 'GITIAN_HOST_IP' in os.environ.keys():
+                os.environ['GITIAN_HOST_IP'] = '10.0.3.1'
+            if not 'LXC_GUEST_IP' in os.environ.keys():
+                os.environ['LXC_GUEST_IP'] = '10.0.3.5'
```

Comment:
These should also be exported on cosmic and later (and maybe debian?)

So no need to guard by `is_bionic`?

Reply:
Fixed